### PR TITLE
PROD-10137: Refactoring nuage heat templates

### PR DESCRIPTION
**Pull request by sunny-verma**
This patch includes:
	1. Moving Nuage VRS/AVRS templates to /home/stack
	2. Removing obselete code from patching script
	3. Support for multirole AVRS starting from 5.4.1u4
	4. Documentation changes:
		* Accomodating above changes
		* PROD-9702: Adding RHSB support
		* Seperating 5.4.1u4 document with earlier version
	5. Using Signed packages support for Undercloud patching
		and Overcloud Deployment
	6. PROD-10132: Solving Issues on github.com
		* issues #68 
		* issues #63
        7. removing unwanted dependecies like  python-novaclient
        8. Fixes: OPENSTACK-2555

### CODE REVIEW

File: nuage-tripleo-heat-templates/docker/services/nova-compute-avrs.yaml [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        puppet_tags: nova_config,nova_paste_api_ini
+        step_config:
+          list_join:
+            - "\n"
+            - - {get_attr: [NovaComputeAvrsBase, role_data, step_config]}
+              - {get_attr: [MySQLClient, role_data, step_config]}
+        config_image: {get_param: DockerNovaLibvirtAvrsConfigImage}
+      kolla_config:
+        get_attr: [NovaComputeAvrsDockerBase, role_data, kolla_config]
+      docker_config_scripts:Making Re
```
Comment (mwhahaha) [inline nuage-tripleo-heat-templates/docker/services/nova-compute-avrs.yaml]: There appears to be some extra items here "Making Re" probably needs to be removed

Comment (sunny-verma) [reply nuage-tripleo-heat-templates/docker/services/nova-compute-avrs.yaml]: Thanks. Updated the PR.

**Comment by jfargen** [comment]
Does the git commit for PR #69 require a specific version of the openstack-tripleo-heat-templates rpm?

**Comment by mwhahaha** [comment]
No it should not require a specific version. The code was specifically updated to leverage existing functionality within THT for OSP13

**Pull request merged**